features/environment.py

```diff
@@ -0,0 +1,58 @@
+import os
+import psycopg2
+
+
+FEATURES_CLEAN_DB = ['aggregator_cleaner', 'aggregator_exporter']
+FEATURES_WITH_KAFKA = []
+FEATURES_WITH_MINIO = ['aggregator_exporter']
+
+
+def before_all(context):
+    context.database_host = os.getenv("DB_HOST", "localhost")
+    context.database_port = os.getenv("DB_PORT", 5432)
+    context.database_name = os.getenv("DB_NAME", "postgres")
+    context.database_user = os.getenv("DB_USER", "postgres")
+    context.database_password = os.getenv("DB_PASS", "postgres")
+    context.kafka_hostname = os.getenv("KAFKA_HOST", "localhost")
+    context.kafka_port = os.getenv("KAFKA_PORT", "9092")
```

Comment:
Will this code define all those values in the context for all the tests, even if they don't use database or Kafka?

Reply:
Yes. But I plan on adding the right filters later when all the features are tagged

Reply:
Well, I trust you :-)

Reply:
Thanks =)